### PR TITLE
Milestone 1: Hello Rails React | Project as one pp with Webpack

### PR DESCRIPTION
**Pull request by samiullah997**
## Milestone 1: Hello Rails React | Project as one App with Webpack
In this Module:
- Create Controllers for APIs and for Root.
- Create Redux and Greeting File for views.
- Generate a Greeting model.
- Generate Migration for Greeting.
- Add Greeting data into the seeds file.
- Update Readme.md file to put project details.

### CODE REVIEW

**Review by rloterh — COMMENTED** [review]
Hi Samiullah, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

Kindly ensure that your ReadMe for this project complies with the following standards:
- [ ] Kindly ensure to remove the template logo. Optionally, you can replace it with your own logo if you have one.
- [ ] Kindly ensure that provided text instructions are removed and filled in with custom content applicable to your project.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: app/javascript/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import { Provider } from 'react-redux'
+
+const App = () => {
+  return (
+      <Provider store={store}>
+          <Greeting />
+      </Provider>
+  )
+}
+
```
Comment (fobadara) [inline app/javascript/App.js]: [REQUIRED] You should have routes that are imported from react-router in app. And you should have a route. leading to Greeting component

**Review by fobadara — CHANGES_REQUESTED** [review]
Hi @samiullah997, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## Highlights
✅ Linters are passing

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @samiullah997, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Clean and organized code

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**